common/backintime.py

```diff
@@ -48,6 +48,14 @@
 
 parsers = {}
 
+def warningOnTakeSnapshot(config):
+    hasMissing, missing = snapshots.hasMissing(config.include())
+    if hasMissing:
+        msgMissing = ', '.join(missing)
+        msg = f'{_("The following folders are missing")}: {msgMissing}'
+        logger.warning(msg)
+    return True
+
```

Comment:
I know that BIT itself do not follow PEP8 coding style guidelines. But we try to do it step by step. I would suggest for new code like you created here use PEP8 rules. For example noCamel case and things like this. It will create style-mixed code but we have to start on some point.

```suggestion
def warning_on_take_snapshot(config):
    has_missing, missing = snapshots.has_missing(config.include())
    
    if has_missing:
        msg = ', '.join(missing)
        msg = f'The following folders are missing: {msg}'
        logger.warning(msg)

    return True

```

Logging output shouldn't be translated. It is not read by regular users. We have to draw a line somewhere between strings to translate and not to translate. When starting with translating log output we would translate every log output. It also makes debugging for us easier if users sending us English terminal output only. Another translation related topic: Always include punctuation marks and similar marks in strings to translate. This is relevant for Right-To-Left or Bidirectional-Languages like Arabic and Hebrew.